src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IArgument.vb

```diff
@@ -746,6 +746,42 @@ BC30057: Too many arguments to 'Public Sub M2(x As Integer, [y As Integer = 0],
 
         <CompilerTrait(CompilerFeature.IOperation)>
         <Fact()>
+        Public Sub TestValidDynamicInvocation_OmittedArgument()
```

Comment:
NOTE: we already have a unit test `Error_ExtraOmittedArgument` just above this one for omitted arguments in error cases.